nixos/tests/additional-python-packages.nix

```diff
@@ -0,0 +1,13 @@
+import ./make-test-python.nix ({ ... }:
+{
+  name = "additional-python-packages";
+
+  withAdditionalPythonPackages = p: [p.numpy];
+
+  nodes.machine = {...}: {};
```

Comment:
Who needs VMs, right?

I guess `nodes = { };` didn't work?

Reply:
No VMs --> probably the fastest nixos test yet! :grin: 

TBH, I didn't try that. It does work---I've fixed it here and in the docs.